src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo:React.FC<Todo> = ({
+  title,
+  completed,
+  user,
+}) => (
+  <>
+    <h2 className="todo__title">{title}</h2>
+    {completed ? (
+      <h3 className="todo__condition-true">Is completed</h3>
+    ) : (
+      <h3 className="todo__condition-false">Is not completed</h3>
+    )}
```

Comment:
better use https://www.npmjs.com/package/classnames

```suggestion
    {completed 
      ? <h3 className="todo__condition-true">Is completed</h3>
      : <h3 className="todo__condition-false">Is not completed</h3>
    }
```

Reply:
i will make like this: https://prnt.sc/26flmaw
because: https://prnt.sc/26flm4q